### DISCUSSION TITLE
Issue with importing Output from pyswmm

### DISCUSSION THREAD

**Discussion by aniroomandi** [pyswmm/pyswmm#487] [Q&A]
Hi,

I just started using pyswmm and installed the package in Anaconda using: conda install -c conda-forge pyswmm

When I run an example (Latte_art.py), I get the following error:

**from pyswmm import Simulation, Nodes, Links, Output
ImportError: cannot import name 'Output' from 'pyswmm' (C:\Users\ ...\Lib\site-packages\pyswmm\__init__.py)**

Could you help resolve this issue please?

Thanks

**Comment by bemcdonnell** [comment]
@aniroomandi Could you please confirm which version of pyswmm you are using?

**Reply by aniroomandi** [reply]
The version is 0.6.2, maybe I should use install the latest version?
![version](https://github.com/pyswmm/pyswmm/assets/38677172/e8c6db79-1098-4362-b7e6-ea25f2c05bee)
